TaskHouseApi/Model/Task.cs

```diff
@@ -15,8 +16,7 @@ public class Task : BaseModel
         public DateTime Deadline { get; set; }
         [Required]
         public string Description { get; set; }
-        [Required]
-        public string Urgency { get; set; }
+        public decimal Urgency { get; set; }
```

Comment:
Does it need to be decimal. Double?

Reply:
Not gonna lie to you hombbre, I have no idea why we used decimal